src/sidebar/store/modules/test/drafts-test.js

```diff
@@ -225,4 +226,44 @@ describe('Drafts Store', () => {
       assert.deepEqual(store.unsavedAnnotations(), []);
     });
   });
+
+  describe('#deleteNewAndEmptyDrafts2', () => {
+    unroll(
```

Comment:
We've been avoiding `unroll` in new code recently in favour of just using an array and `forEach`:

```js
[{
  // Object with properties for test case
}].forEach(({ prop1, prop2, ... }) => {
 it(/* description */, () => {
    // Test case
 });
});
```

The rationale was that not using a custom utility saved developers from having to learn an extra thing, and it made it easier to achieve consistency across projects.